plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimatorsFactory.kt

```diff
@@ -51,7 +51,7 @@ class CameraAnimatorsFactory internal constructor(mapDelegateProvider: MapDelega
       )
     }
     cameraOptions.bearing?.let {
-      var startBearing = currentCameraOptions.bearing ?: it
+      var startBearing = currentCameraState.bearing ?: it
```

Comment:
`?: it` shouldn't be needed anymore, right?

Reply:
True! Thanks for catching!